internal/services/formatters/go/gosec/formatter_test.go

```diff
@@ -24,112 +24,184 @@ import (
 
 	"github.com/ZupIT/horusec-devkit/pkg/entities/analysis"
 	"github.com/ZupIT/horusec-devkit/pkg/enums/tools"
-	cliConfig "github.com/ZupIT/horusec/config"
-	"github.com/ZupIT/horusec/internal/entities/workdir"
+	"github.com/ZupIT/horusec/config"
 	"github.com/ZupIT/horusec/internal/services/docker"
 	"github.com/ZupIT/horusec/internal/services/formatters"
 )
 
-func TestGoLang_StartAnalysis(t *testing.T) {
-	t.Run("Should run analysis and return error because output is empty", func(t *testing.T) {
-		dockerAPIControllerMock := &docker.Mock{}
-		dockerAPIControllerMock.On("SetAnalysisID")
-		dockerAPIControllerMock.On("CreateLanguageAnalysisContainer").Return("", nil)
+func TestGosecStartAnalysis(t *testing.T) {
+	t.Run("Should run analysis with sucessfull when output is empty", func(t *testing.T) {
+		docker := &docker.Mock{}
```

Comment:
Docker variable contains the same imported package name.

Reply:
I don't see this as problem but I change.

Reply:
Variables and packages with the same name may cause confusion. For example, trying to call a package function but it's calling the variable instead or inverse.